cassandra.cluster can not support multiple IPs

## what I do
I run go-ycsb to test cassandra by multiple cassandra endpoints
```shell
./bin/go-ycsb load cassandra -p recordcount=1000000 -p cassandra.cluster=127.0.0.1:9042,127.0.0.2:9042 -p mysql.port=4000
```
## what happened
```
panic: runtime error: index out of range

goroutine 1 [running]:
main.initialGlobal(0x7ffeefbff953, 0x5, 0xc000373c18)
	/Users/xiaojingchen/go/src/github.com/pingcap/go-ycsb/cmd/go-ycsb/main.go:91 +0x93e
main.runClientCommandFunc(0xc0003de500, 0xc0003f8070, 0x1, 0x7, 0x0)
	/Users/xiaojingchen/go/src/github.com/pingcap/go-ycsb/cmd/go-ycsb/client.go:30 +0xbb
main.runLoadCommandFunc(0xc0003de500, 0xc0003f8070, 0x1, 0x7)
	/Users/xiaojingchen/go/src/github.com/pingcap/go-ycsb/cmd/go-ycsb/client.go:62 +0x4e
github.com/spf13/cobra.(*Command).execute(0xc0003de500, 0xc0003f8000, 0x7, 0x7, 0xc0003de500, 0xc0003f8000)
	/Users/xiaojingchen/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003de000, 0xc000373f30, 0x3, 0x3)
	/Users/xiaojingchen/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/xiaojingchen/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main()
	/Users/xiaojingchen/go/src/github.com/pingcap/go-ycsb/cmd/go-ycsb/main.go:170 +0x292
```
## The reason
the function `StringSliceVarP` can not parse the values which contains `,`correctly.
e.g: `-p a=b,c` was parsed as `a=b` and `c`
```go
// StringSliceVar defines a string flag with specified name, default value, and usage string.
// The argument p points to a []string variable in which to store the value of the flag.
// Compared to StringArray flags, StringSlice flags take comma-separated value as arguments and split them accordingly.
// For example:
//   --ss="v1,v2" -ss="v3"
// will result in
//   []string{"v1", "v2", "v3"}
func (f *FlagSet) StringSliceVar(p *[]string, name string, value []string, usage string) {
	f.VarP(newStringSliceValue(value, p), name, "", usage)
}

// StringSliceVarP is like StringSliceVar, but accepts a shorthand letter that can be used after a single dash.
func (f *FlagSet) StringSliceVarP(p *[]string, name, shorthand string, value []string, usage string) {
	f.VarP(newStringSliceValue(value, p), name, shorthand, usage)
}

```

--- a/.gitignore
+++ b/.gitignore
@@ -2,3 +2,4 @@ bin
 data
 logs
 log
+.idea

--- a/cmd/go-ycsb/client.go
+++ b/cmd/go-ycsb/client.go
@@ -73,7 +73,7 @@ var (
 
 func initClientCommand(m *cobra.Command) {
 	m.Flags().StringSliceVarP(&propertyFiles, "property_file", "P", nil, "Spefify a property file")
-	m.Flags().StringSliceVarP(&propertyValues, "prop", "p", nil, "Specify a property value with name=value")
+	m.Flags().StringArrayVarP(&propertyValues, "prop", "p", nil, "Specify a property value with name=value")
 	m.Flags().StringVar(&tableName, "table", "", "Use the table name instead of the default \""+prop.TableNameDefault+"\"")
 	m.Flags().IntVar(&threadsArg, "threads", 1, "Execute using n threads - can also be specified as the \"threadcount\" property")
 	m.Flags().IntVar(&targetArg, "target", 0, "Attempt to do n operations per second (default: unlimited) - can also be specified as the \"target\" property")

